src/lib/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface Room {
+  id: string;
+  name: string;
+}
+
+export interface Event {
+  id: string;
+  status: EventStatus;
+  title: string;
+  subject: string;
+  description: string;
+  speaker: string;
+  roomId: string;
+  startTime: string;
+  endTime: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+export type EventStatus = "PENDING_APPROVAL" | "APPROVED" | "REJECTED";
```

Comment:
Histoire de match avec le back, on peut éventuellement rester sur "**Talk**", tu peux aussi rajouter le **level** et le **subject** ;)

Tu mets pas **createdAt** et **updatedAt** en type Date ?

```suggestion
export interface Talk {
  id: string;
  status: TalkStatus;
  title: string;
  subject: TalkSubject;
  description: string;
  speaker: string;
  roomId: string;
  startTime: string;
  endTime: string;
  level: TalkLevel;
  createdAt: Date;
  updatedAt: Date;
}

export type TalkStatus = "PENDING_APPROVAL" | "APPROVED" | "REJECTED";

export type TalkSubject = "AI" | "WEB_DEVELOPMENT" | "MOBILE_DEVELOPMENT" | "DATA_SCIENCE" | "CLOUD_COMPUTING" | "CLOUD_COMPUTING" | "CYBER_SECURITY" | "CYBER_SECURITY" | "CYBER_SECURITY" | "CYBER_SECURITY";

export type TalkLevel = "BEGINNER" | "INTERMEDIATE" | "ADVANCED";
```